exercises/RobertoAssandri/TheTeenager/TheTeenagerProgram.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+const questionMark = "?"
+
+const question = "Sure."
+const yell = "Whoa, chill out!"
+const yellQuestion = "Calm down, I know what I'm doing!"
+const addressHim = "Fine. Be that way!"
+const everythingElse = "Whatever."
+
+const questionType = 1
+const yellType = 2
+const yellQuestionType = 3
+const addressHimType = 4
+const everythingElseType = 5
+
+func AnalyzePhrase(phrase string) int{
+	if phrase == ""{
+		return addressHimType
+	}
+
+	lastCharacterStr := string(phrase[len(phrase)-1])
```

Comment:
Question condition could be supported by char question mark instead of string:
const questionMark = '?'
...
lastCharacter := phrase[len(phrase)-1]
..